system-int-test/test/cmr/system_int_test/search/autocomplete/suggestion_reindex_test.clj

```diff
@@ -76,8 +75,26 @@
                             (fu/processing-level-id "PL1")
                             {:DataCenters [(data-umm-spec/data-center {:Roles ["ARCHIVER"] :ShortName "DOI/USGS/CMG/WHSC"})]
                              :ScienceKeywords [(:ScienceKeywords (fu/science-keywords sk1 sk2))]})
+        coll3 (d/ingest-concept-with-metadata-file "CMR-6287/C1000000029-EDF_OPS.xml"
+                                                         {:provider-id "PROV1"
+                                                          :concept-type :collection
+                                                          :format-key :echo10})
         _ (index/wait-until-indexed)]
 
     (index/reindex-suggestions)
     (index/wait-until-indexed)
-    (search/get-autocomplete-suggestions "q=level2")))
+    (testing "Ensure that response is in proper format and results are correct"
+      (is (= (get-in (search/get-autocomplete-json "q=level2") [:feed :entry])
+            [{:score 0.25604635 :type "organization" :value "Langley DAAC User Services" :fields "Langley DAAC User Services"}
```

Comment:
I am guilty of hard-coding these as well, but do we have a "near" operator for float comparison?

Reply:
Not in the way that you're thinking of as far as I know. We have a way to compare collections such that the order of the items doesn't matter, but that's not really useful here. I assume that if the data never changes then none of the results should either

Reply:
Does the actual score value matter here? Can you just do two assertions, one with the fields you expect and one making sure the scores are in descending order. This seems like it could easily change the next time you add tests to it.

Reply:
Seems like you could just test for the existence of the `score`.